.github/workflows/test-harness.yml

```
@@ -14,3 +14,4 @@ jobs:
         with:
           sdks-to-test: ruby
           sdk-github-sha: ${{github.event.pull_request.head.sha}}
+          sdk-capabilities: '{ "Ruby": ["clientCustomData", "v2Config", "variableFeatureId", "allVariables", "allFeatures", "evalReason", "eventsEvalReason", "variablesFeatureId"]}'
```

nitpick: not impacting this PR, but invalid entry of `variableFeatureId`, should only have the plural one which is included at the end already. 
```suggestion
          sdk-capabilities: '{ "Ruby": ["clientCustomData", "v2Config", "allVariables", "allFeatures", "evalReason", "eventsEvalReason", "variablesFeatureId"]}'
```